### ISSUE TITLE
Fix "TODO" issue in packages/fast-components-styles-msft/src/heading/index.ts

### ISSUE THREAD

**Issue by awentzel** [Microsoft/fast-dna#315]
TODO found

https://codeclimate.com/github/Microsoft/fast-dna/packages/fast-components-styles-msft/src/heading/index.ts#issue_5adfa15298a8bf00010001d9

**Comment by chrisdholt** [comment]
related to #306

**Comment by chrisdholt** [comment]
Blocked by #306

**Comment by Falkicon** [comment]
This task should be updated to reflect the work that needs to be done. Moving back to triage till updated with proper description.

**Comment by chrisdholt** [comment]
@Falkicon Looking back through, this technically isn't blocking #306 but a dupe of #306. The todo references the task directly. Suggest we close this in favor of #306.

Additionally, I think there is a utility somewhere linked up in github that automatically adds tasks for TODO's found in code. The problem with that is our policy is to reference an issue with the TODO, so many of these are likely dupes.

**Comment by chrisdholt** [comment]
closing in favor of #306
